compiler/rustc_llvm/llvm-wrapper/RustWrapper.cpp

```diff
@@ -983,6 +983,9 @@ extern "C" LLVMMetadataRef LLVMRustDIBuilderCreateStaticMemberType(
     unwrapDI<DIType>(Ty),
     fromRust(Flags),
     unwrap<llvm::ConstantInt>(val),
+#if LLVM_VERSION_GE(17, 0)
```

Comment:
Since this is new as of ~yesterday, shouldn't this be checking for LLVM >= 18, not 17?

Reply:
done